packages/api-core/src/site-scan/fetchers.ts

```diff
@@ -1,10 +1,10 @@
 import { wpcom } from '../wpcom-fetcher';
-import type { SiteScan, SiteScanHistory } from './types';
+import type { SiteScan, SiteScanHistory, FixThreatsStatusResponse } from './types';
 
 export async function fetchFixThreatsStatus(
 	siteId: number,
 	threatIds: number[]
-): Promise< SiteScan > {
+): Promise< FixThreatsStatusResponse > {
```

Comment:
Replaced `SiteScan` with the new `FixThreatsStatusResponse` because the responses are not the same.